### ISSUE TITLE
Tree style tab just doesn't work at all anymore

### ISSUE THREAD

**Issue by ksze** [piroor/treestyletab#1755]
## Short description
Tree style tab doesn't work after updating to version 2.4.8.

## Steps to reproduce
NOTE: starting with a clean profile does not apply because this problem happens when **updating** the extension.
 1. Start an existing Firefox with an existing profile with some existing windows and tabs, and existing TST (I'm not sure if I had 2.4.6 or 2.4.7 prior to updating)
 2. Update TST to 2.4.8
 3. The TST extension automatically reloads.

## Expected result
TST reloads in a working state, with my tree restored.

## Actual result
The TST sidebar opens, but it's just a white dot bouncing left and right on a gray background.
- Close the sidebar and reopen --> still the same bouncing white dot.
- Disable and re-enable the extension --> still the same bouncing white dot.
- Close and reopen Firefox --> still the same bouncing white dot.

## Environment

 * Platform (OS): Windows 10 Professional 64-bit Fall Creator Update Version 1709 (build 16299.192)
 * Version of Firefox: Firefox 58.0.1 64-bit
 * Version (or revision) of Tree Style Tab: 2.4.8

**Comment by ksze** [comment]
More details. It seems that TST 2.4.8 has broken behaviour with regards to container tabs. If I disable these extensions:
- Firefox Multi-Account Containers 5.0.1
- Context Plus 0.5.0

Then TST can reload. **HOWEVER** it loses all my windows and containered tabs. Only a very limited number of them can be salvaged from Firefox's "Recently closed tabs" and "Recently closed windows".

**Comment by DerGenaue** [comment]
Possible duplicate of #1735?

**Comment by ksze** [comment]
Maybe. Even the workaround described in #1735 seems not very reliable though; I had to retry it a couple of times for the tree to load. And then I get these ugly, red-dot borders around the elements in the tree. Is there a way I can get rid of those borders?

**Comment by piroor** [comment]
I've tried with these steps:

 * Setup Firefox (Nightly 60.0a1)
 * Install TST 2.4.7, FIrefox Multi-Account Containers 5.0.1, and Context Plus 0.5.0.
 * Update TST to 2.4.8

But I couldn't reproduce any problem. There are possibly some more triggers around configs of those addons. @ksze Did you check "Always open in Default" for any domain, at the popup menu of Firefox Multi-Account Containers?

**Comment by ksze** [comment]
@piroor No, I don't have "Always open in Default" (or "Always open in <any container>") checked for any domain.

**Comment by Zefling** [comment]
It's pretty recurrent, infinite loading like that.
![tst_loading](https://user-images.githubusercontent.com/3032555/40230507-0cc7f6b2-5a98-11e8-9f1d-069423ba5916.png)

**Comment by piroor** [comment]
@RanHum Thanks a lot! Your log is quite similar to other logs from contributors, so they all may see same problem. Now this issue is quite same to #1735, so I close this as a duplicated. For further research, could you cooperate to collect detailed logs on the startup process? Steps to collect logs are:

 1. Install debug build of TST: https://piro.sakura.ne.jp/xul/xpi/nightly/treestyletab-we-debug.xpi
 2. Restart Firefox to reproduce the problem.
 3. Start remote debugger for TST (instructions: https://github.com/piroor/treestyletab/wiki/How-to-inspect-tree-of-tabs#how-to-collect-debug-logs-of-tst )
 4. Choose `/sidebar/sidebar.html?` from the namespace selector of the remote debugger, and run a script `log.logs.join('\n')` in the console. Then you'll see recent 1000 logs, so please paste it to #1735.
